esp32_ulp/preprocess.py

```diff
@@ -32,29 +32,31 @@ def WRITE_RTC_FIELD(rtc_reg, low_bit, value):
 
 class Preprocessor:
     def __init__(self):
+        self._defines_db = None
         self._defines = {}
 
+    def parse_define_line(self, line):
+        line = line.strip()
+        if not line.startswith("#define"):
+            # skip lines not containing #define
+            return {}
+        line = line[8:].strip()  # remove #define
+        parts = line.split(None, 1)
+        if len(parts) != 2:
+            # skip defines without value
+            return {}
+        identifier, value = parts
+        tmp = identifier.split('(', 1)
+        if len(tmp) == 2:
```

Comment:
can it be > 2?

Reply:
No, because of the `,1` argument to the split function, which means only split once. This results in either a single value (if there was no way to split) or 2 values (if there was), but never more.

Reply:
ah, right, sorry, overlooked that.